libbeat/outputs/elasticsearch/enc.go

```diff
@@ -89,8 +89,8 @@ func (b *jsonEncoder) resetState() {
 
 	b.folder, err = gotype.NewIterator(visitor,
 		gotype.Folders(
-			codec.MakeTimestampEncoder(),
```

Comment:
@ruflin @urso  This is old milliseconds encoder, now i remove it. many testing failed. So i am not sure that is it this change correct?